scripts/pds-ingress-client.py

```diff
@@ -6,14 +6,15 @@
 import requests
```

Comment:
This file could be in the src folder with a binary command generated when pip install runs. Is there a reason why it is not the case. Maybe because this is the only client side part in the development...

Reply:
I think I set it up this way out of habit, as I'm not as familiar with the approach you mentioned.